clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/provider/config/CloudFoundryProviderConfig.java

```diff
@@ -78,7 +99,9 @@ public ForkJoinPool cloudFoundryThreadPool(
                 cacheRepository,
                 a.getPermissions().build(),
                 cloudFoundryThreadPool,
-                a.getSpaceFilter()),
+                a.getSpaceFilter(),
+                new OkHttpClient(),
```

Comment:
We should be using the new okhttpclient from the method above. Check line 80 as we are already autowiring it as a method parameter

Reply:
Changed!